swarooppatilx/scruter
# Hower Effect Added On Links #85

What does this PR do?
This PR adds hover effects to the social media links to enhance the user experience. When users hover over the links, the color will change to match the branding of each social media platform, providing a visual cue.

Fixes #85

Type of change
 New feature (non-breaking change which adds functionality)
How should this be tested?
 Hover over the social media links to ensure that the hover effect is working correctly for each platform (e.g., Facebook, Twitter, LinkedIn).
 Test responsiveness and check if the hover effects apply correctly on both desktop and mobile views.
